lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -593,6 +593,8 @@ def get_client(self, job_destination_params, job_id, env=None):
             files_endpoint=files_endpoint,
             env=env
         )
+        # Turn MutableDict into standard dict for pulsar consumption
+        job_destination_params = dict(job_destination_params.items())
```

Comment:
So this could be a bugfix ... or not, but probably it is a bugfix. This used to be a a mutable column, so when in the pulsar client we do https://github.com/galaxyproject/pulsar/blob/68a52084468b48a447a4849498d30ac69b805900/pulsar/client/manager.py#L87 we actually change the job_destination_params in the database.

Reply:
I did know it updated those, Nate and I hit that multiple times last week. I don't know if we're dependent on those database parameters being updated or not (past John didn't leave any clues why he did this xref https://github.com/galaxyproject/pulsar/commit/e4aeb22bca67ab3d20555b81fa73b33a8390d0f9).

I'm okay with merging this and seeing if something breaks. It should be pretty clear if files_endpoint or something needs to be populated initially for updates to work or something odd like that.